tests/runfiles/common.run

```diff
@@ -411,12 +404,6 @@ tests = ['zpool_remove_001_neg', 'zpool_remove_002_pos',
     'zpool_remove_003_pos']
 tags = ['functional', 'cli_root', 'zpool_remove']
 
-[tests/functional/cli_root/zpool_reopen]
-tests = ['zpool_reopen_001_pos', 'zpool_reopen_002_pos',
-    'zpool_reopen_003_pos', 'zpool_reopen_004_pos', 'zpool_reopen_005_pos',
-    'zpool_reopen_006_neg', 'zpool_reopen_007_pos']
-tags = ['functional', 'cli_root', 'zpool_reopen']
```

Comment:
Is it necessary to move all of these tests?  The bulk of the `scsi_debug` functionality should already be contained in wrapper functions provided by `blkdev.shlib`, and tests which depend on it should already be skipped when it's unavailable.

```sh
# Create scsi_debug devices for the reopen tests
if is_linux; then
        load_scsi_debug $SDSIZE $SDHOSTS $SDTGTS $SDLUNS '512b'
else
        log_unsupported "scsi debug module unsupported"
fi
```

Could we instead make this check more generic, so that when you do implement this functionality the tests will automatically work. Maybe something like:

```sh
if ! have_blkdev_debug; then
        log_unsupported "Debug block devices unsupported"
fi
```

Reply:
Skipped tests complicate zts-report.py. I was hoping to avoid that by moving the tests out of the common run file until they are functional on platforms other than Linux, but it makes sense to leave them and *expect* platforms to provide implementations for the scsi debug parts. I will put the scsi debug tests back for this PR and rework the check separately.

Reply:
> Skipped tests complicate zts-report.py.

That makes sense.  We'll have to give some thought as to how we want the ` zts-report.py` script to work in a multi-platform world.

> I will put the scsi debug tests back for this PR and rework the check separately.

Sounds good.

Reply:
Thanks, this LGTM we'll just wait on the bots.